backend/controllers/message_controller.js

```diff
@@ -13,29 +13,22 @@ const Message = require("../models/Message.js");
 // 7. If saving succeeds, call `addMessageToChat` to associate the message with
 //    the chat.
 const create_message = async (request, response) => {
-  try {
+  
     const chat_id = extract_chat_id(request);
     const user_id = extract_user_id(request);
     const message = extract_message(request);
 
     const input_status = await validate_inputs(chat_id, user_id, message);
     if (input_status.status == 400) {
-      respond_with_error_json(response, input_status);
```

Comment:
Just put this back then throw an error after it runs.

Reply:
So that the client knows it fails from the response and message_post knows it fails from the error you throw after.